wandb/apis/public.py

```diff
@@ -4312,10 +4318,19 @@ def from_id(cls, artifact_id: str, client: Client):
                             )
                             break
 
+            entity, project = None, None
+            if response["artifact"]["artifactType"] is not None:
```

Comment:
style nit - if you want to avoid all the conditionals. You can do
project = response.get("artifact", {}).get("artifactType", {}).get("project")
project_name = project.get("name") # defaults to None
entity_name = project.get("entity", {}).get("name")